src/main/scala/org/broadinstitute/dsde/workbench/sam/config/AppConfig.scala

```diff
@@ -148,25 +151,37 @@ object AppConfig {
     LiquibaseConfig(config.getString("changelog"), config.getBoolean("initWithLiquibase"))
   }
 
+  final case class AdminConfig(superAdminsGroup: WorkbenchEmail, adminEmailDomains: Set[String])
+
+  implicit val adminConfigReader: ValueReader[AdminConfig] = ValueReader.relative { config =>
+    AdminConfig(
+      superAdminsGroup = WorkbenchEmail(config.getString("superAdminsGroup")),
+      adminEmailDomains = config.as[Set[String]]("emailDomains")
```

Comment:
I would call this `allowedAdminEmailDomains` in the config file

Reply:
Good idea